src/Apps/Artist/Routes/AuctionResults/TableColumns.tsx

```diff
@@ -1,17 +1,21 @@
 import { Sans } from "@artsy/palette"
 import React from "react"
 import { Col } from "Styleguide/Elements/Grid"
-import { Responsive } from "Utils/Responsive"
+import { Media } from "Utils/Responsive"
 
 export const TableColumns = () => {
   return (
-    <Responsive>
-      {({ xs, sm, md }) => {
-        if (xs) return null
-        else if (sm || md) return <SmallTableColumns />
-        else return <LargeTableColumns />
-      }}
-    </Responsive>
+    <>
+      <Media at="sm">
+        <SmallTableColumns />
+      </Media>
+      <Media at="md">
```

Comment:
Could this not just be `<Media between={["sm", "md"]}>`? Or, like @damassi recommended above `<Media at={["sm", "md"]}>`

Reply:
Oh yeah, absolutely! 👍  Hah, I missed that they were rendering the same component.